fhir-parent/pom.xml

```diff
@@ -558,7 +558,7 @@
             <dependency>
                 <groupId>com.azure</groupId>
                 <artifactId>azure-storage-blob</artifactId>
-                <version>12.6.0</version>
+                <version>12.13.0</version>
```

Comment:
Hi Martin, 

Please note Lee merged the fix for the 12.6.0 being removed from OSSRH. 

https://github.com/IBM/FHIR/blob/main/fhir-parent/pom.xml#L561
```
            <dependency>
                <groupId>com.azure</groupId>
                <artifactId>azure-storage-blob</artifactId>
                <version>12.13.0</version>
            </dependency>
```
This is already

Reply:
If you Fetch and Merge on the main branch, it'll sync you up with the change. 
![image](https://user-images.githubusercontent.com/3016328/129721857-f3f1fb5b-6126-4c2e-96d7-e70a4e0c5030.png)